pkg/test/template/util_test.go

```diff
@@ -0,0 +1,75 @@
+package template
+
+import (
+	"testing"
+
+	"github.com/codeready-toolchain/member-operator/pkg/apis"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func TestDecodeTemplate(t *testing.T) {
+	s := scheme.Scheme
+	err := apis.AddToScheme(s)
+	require.NoError(t, err)
+	codecFactory := serializer.NewCodecFactory(s)
+	decoder := codecFactory.UniversalDeserializer()
+
+	t.Run("valid_template", func(t *testing.T) {
+		// test
+		tmpl, err := DecodeTemplate(decoder, validTmplContent)
+
+		assert.NoError(t, err)
```

Comment:
you should use 
```suggestion
		require.NoError(t, err)
``` 
here instead